sdk/storage/AzureStorageBlob/Source/Operation/ResumableOperation.swift

```diff
@@ -63,6 +64,7 @@ internal class ResumableOperation: Operation {
     open override func main() {
         if isCancelled || isPaused { return }
         internalState = .complete
+        super.main()
```

Comment:
Not sure if this was contributing to the problem, but best practice, this should call its superclass like the other methods do.